ci-operator/step-registry/hypershift/kubevirt/install/odf/hypershift-kubevirt-install-odf-commands.sh

```diff
@@ -23,56 +23,6 @@ metadata:
   name: "${ODF_INSTALL_NAMESPACE}"
 EOF
 
-# TODO remove this override once https://issues.redhat.com/browse/CLOUDDST-18990 is resolved
-# ODF isn't in the 4.14 catalog, which causes the install to fail. This workaround
-# should work for both 4.13 and 4.14, which are the only two versions being tested
-# at this point in time.
-#
-# Override the subscription source
-ODF_SUBSCRIPTION_SOURCE="redhat-operators-4-13"
-# create the custom catalog source that points to 4.13 regardless of the OCP version
-oc apply -f - <<EOF
-apiVersion: operators.coreos.com/v1alpha1
-kind: CatalogSource
-metadata:
-  annotations:
-    operatorframework.io/managed-by: marketplace-operator
-    target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
-  generation: 5
-  name: redhat-operators-4-13
-  namespace: openshift-marketplace
-spec:
-  displayName: Red Hat Operators
-  grpcPodConfig:
-    nodeSelector:
-      kubernetes.io/os: linux
-      node-role.kubernetes.io/master: ""
-    priorityClassName: system-cluster-critical
-    securityContextConfig: restricted
-    tolerations:
-    - effect: NoSchedule
-      key: node-role.kubernetes.io/master
-      operator: Exists
-    - effect: NoExecute
-      key: node.kubernetes.io/unreachable
-      operator: Exists
-      tolerationSeconds: 120
-    - effect: NoExecute
-      key: node.kubernetes.io/not-ready
-      operator: Exists
-      tolerationSeconds: 120
-  icon:
-    base64data: ""
-    mediatype: ""
-  image: registry.redhat.io/redhat/redhat-operator-index:v4.13
-  priority: -100
-  publisher: Red Hat
-  sourceType: grpc
-  updateStrategy:
-    registryPoll:
-      interval: 10m
-EOF
-
```

Comment:
can we handle what to do about this work around in another PR? I want to make sure we give this some more focused attention.

This workaround existed because ODF was late to publishing their operator to the latest operator catalog. it's possible that the issue this workaround perviously solved is still present for future versions.. for example, it's possible that 4.17 still doesn't have ODF and we'd have to use the 4.16 catalog to make ODF work.

Reply:
there was an issue in deploying ODF in the early revolutions for this PR.
i reverted all odf-related changes, we'll re-evaluate.
we'll open a dedicated PR for ODF configurations across our lanes.